index.js

```diff
@@ -35,12 +36,44 @@ function getConfigs() {
     skipTags: devEnv
       ? yn(process.env.SKIP_TAGS)
       : yn(core.getInput("skip-tags")),
+    retry: true,
   };
 }
 
+const ThrottledOctokit = Octokit.plugin(throttling);
+
 async function run() {
   const configs = getConfigs();
-  const octokit = new Octokit();
+  const octokit = new ThrottledOctokit({
+    throttle: {
+      onRateLimit: (retryAfter, options) => {
+        console.error(
+          `Request quota exhausted for request ${options.method} ${options.url}, remaining requests: ${options.request.retryCount}`
+        );
+
+        if (options.request.retryCount === 0) {
```

Comment:
Let's set the maximum number of retries to ~3.

Reply:
5369a8b

Reply:
This is not needed anymore as retries are global :(

Reply:
Yea, but we'd like to keep the retries as-is without the conditional, right?

Reply:
188c3bc

Reply:
Yes :)